src/components/Header/index.tsx

```diff
@@ -0,0 +1,252 @@
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		position: fixed;
+		background: white;
+		top: 0;
+		right: 0;
+		width: 300px;
+		bottom: 0;
+		justify-content: flex-start;
+		transform: translateX(100%);
+		transition: transform 200ms ease-in-out;
+		padding: 16px;
+		overflow: auto;
+		flex-wrap: nowrap;
+
+		${(props) =>
+			props.menuOpen
+				? css`
+						position: fixed;
+						transform: translateX(0%);
+				  `
+				: ''}
+	}
+`;
+
+export const Navbar = styled(Flex)`
+	min-height: 80px;
+	position: relative;
+	flex-direction: column;
+	justify-content: center;
+
+	@media (max-width: ${(props) =>
+			invertBreakpoint(props.theme.breakpoints[1])}px) {
+		flex-direction: row;
+		justify-content: space-between;
+		align-items: center;
+		min-height: auto;
+		padding-top: 16px;
+		padding-bottom: 16px;
+	}
+`;
+
+export const Header = (props: HeaderProps) => {
+	const [mobileMenuOpen, setMobileMenuOpen] = React.useState(false);
+
+	const closeMenu = () => {
+		setMobileMenuOpen(false);
+	};
+	const openMenu = () => {
+		setMobileMenuOpen(true);
+	};
+	return (
+		<Wrapper>
+			<Container px={[3, 3, 3, 0]}>
+				<Navbar>
+					<Flex justifyContent="space-between" alignItems="center" width="100%">
+						<Flex alignItems="center">
+							<Link href={props.rootRoute} style={{ lineHeight: 1 }}>
+								<Img alt={props.logoAltText} src={props.logo} height="42" />
+							</Link>
+						</Flex>
+						{mobileMenuOpen && (
+							<Fixed
+								top
+								left
+								bottom
+								display={['block', 'block', 'none', 'none']}
+								right
+								bg="rgba(0, 0, 0, 0.25)"
+								onClick={closeMenu}
+							/>
+						)}
+						<Flex>
+							<NavigationItems
+								menuOpen={mobileMenuOpen}
+								flexDirection={['column', 'column', 'row', 'row']}
+								flexWrap="wrap"
+								alignItems={['flex-start', 'flex-start', 'center', 'center']}
+							>
+								{mobileMenuOpen && (
+									<Flex
+										display={['block', 'block', 'none', 'none']}
+										justifyContent="flex-end"
+									>
+										<Button
+											plain
+											underline={false}
+											p="5px"
+											mt="-8px"
+											mr="-5px"
+											style={{ opacity: 0.5 }}
+											icon={<FontAwesomeIcon icon={faTimes} />}
+											onClick={closeMenu}
+										/>
+									</Flex>
+								)}
+								{getNavigationRoutes(props.routes)}
+
+								{props.actions && (
+									<MobileActions
+										mt="auto"
+										pt={5}
+										alignItems="center"
+										justifyContent="center"
+									>
+										{props.actions}
+									</MobileActions>
+								)}
+							</NavigationItems>
+							{props.actions && (
+								<Actions display={['none', 'none', 'flex', 'flex']} ml={3}>
+									{props.actions}
+								</Actions>
+							)}
+						</Flex>
+						<Button
+							display={['block', 'block', 'none', 'none']}
+							style={{ fontSize: 20 }}
```

Comment:
Would applying the fontSize on the icon itself, or on the button using `fontSize` prop work?